lib/anyway/config.rb

```diff
@@ -155,5 +190,24 @@ def parse_yml(path)
         YAML.load_file(path)
       end
     end
+
+    def build_option_parser
+      require 'optparse'
+      @option_parser ||= OptionParser.new do |opts|
+        self.class.option_parser_attributes.each do |(key, options)|
```

Comment:
What is `options`? Isn't `option_parser_attributes` an array of key?

Reply:
It's a leftover from the previous implementation. Initially, I wanted to store the `option_parser_attributes` as a hash, like
```ruby
{
  host: { description: 'Host name' },
  ...
}
```

Then I've changed my mind but didn't update the code accordingly.